CodeStarterCamp_Week2/main.swift

```diff
@@ -6,28 +6,28 @@
 //  Copyright © yagom academy. All rights reserved.
 //
 
+var Round: Int = 0
+var roundNumbers: Dictionary<String, Set<Int>> = [String: Set<Int>]()
```

Comment:
```suggestion
var roundNumbers: [String: Set<Int>] = .init()
```
요즘 컨벤션은 Dictionary를 명시하지 않는 걸로 알고 있어요 ㅎㅎ